scripts/qa_crowbarsetup.sh

```diff
@@ -1747,7 +1753,6 @@ function reboot_nodes_via_ipmi
                 "default gateway to be active in bmc"
 
             $ipmicmd chassis bootdev pxe options=persistent
-            $ipmicmd mc reset warm
```

Comment:
@dirkmueller @kbaikov 
ok, fine, but then if the `reset warm` goes, then the next two wait_for's are useless, should go as well